src/humanPose/index.js

```diff
@@ -213,12 +326,42 @@ import * as utils from './utils.js'
     }
 
     // initialize the human pose object
-    function handleDiscoveredObject(object, objectKey) {
+    function handleDiscoveredHumanPose(object, objectKey) {
         if (!utils.isHumanPoseObject(object)) { return; }
         if (typeof humanPoseObjects[objectKey] !== 'undefined') { return; }
         humanPoseObjects[objectKey] = object; // keep track of which human pose objects we've processed so far
 
-        // TODO: subscribe to public data, etc
+
+        if (objectKey === myHumanPoseId) {
+            // no action for now
+        } else {
+            // subscribe to public data of a selected joint node in remote HumanPoseObject which transfers whole pose
+            let keys = utils.getJointNodeInfo(object, 0);
+            if (!keys) { return; }
+
+            let subscriptionCallback = (msgContent) => {
+                // update public data of node in local human pose object
+                let node = realityEditor.getNode(msgContent.object, msgContent.frame, msgContent.node);
+                if (!node) { 
+                    console.warn('couldn\'t find the node ' + msgContent.node + ' which stores whole pose data');
+                    return; 
+                }
+                node.publicData[utils.JOINT_PUBLIC_DATA_KEYS.transferData] = msgContent.publicData.whole_pose;
```

Comment:
Not sure if this would be cleaner as `node.publicData[utils.JOINT_PUBLIC_DATA_KEYS.transferData] = msgContent.publicData[utils.JOINT_PUBLIC_DATA_KEYS.transferData];` – I had to look up whole_pose to see where it was coming from